client/src/components/Comment.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FC, useState } from "react";
+import { INewsId } from "../types/INewsId";
+import { CaretDownOutline, CaretUpOutline } from "react-ionicons";
+
+interface CommentProps {
+  comment: INewsId;
+  disable?: boolean;
+}
+
+const Comment: FC<CommentProps> = ({ comment, disable }) => {
+  const [visible, setVisible] = useState<boolean>(true);
```

Comment:
explicit тут необязательно задавать, ts и так поймет
```suggestion
  const [visible, setVisible] = useState(true);
```

Reply:
Та в курсе, скорее дело привычки, поэтому написал, надо избавляться.